package.json

```diff
@@ -17,36 +17,35 @@
     "babel-preset-react": "^6.5.0",
     "css-loader": "^0.25.0",
     "eslint": "^5.2.0",
-    "eslint-plugin-import": "^2.16.0",
-    "eslint-plugin-node": "^7.0.1",
     "eslint-config-standard": "^11.0.0",
+    "eslint-plugin-import": "^2.19.1",
+    "eslint-plugin-node": "^7.0.1",
     "eslint-plugin-promise": "^3.8.0",
-    "eslint-plugin-react": "^7.10.0",
+    "eslint-plugin-react": "^7.17.0",
     "eslint-plugin-standard": "^3.1.0",
-    "file-loader": "^0.9.0",
     "font-awesome": "^4.6.1",
     "gh-pages": "^0.11.0",
-    "jasmine-core": "^2.2.0",
-    "jquery": "^2.2.4",
-    "karma": "^1.3.0",
+    "jasmine-core": "^2.99.1",
+    "jquery": "^3.4.1",
```

Comment:
Please also update the peerDependency https://github.com/livingdocsIO/editable.js/pull/174/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L68
And then this could also be a breaking change... depending on what range we declare in there.
Do we drop support for jquery 2?

Reply:
Would that be a breaking change from the editable perspective? Or do peer-dependencies take either 3.x.x and then if that's not available they take what's available so 2.x.?

Reply:
Seems so, in this case I marked it as breaking change.

Jquery 2.x.x should still work, we just won't actively support it IMO.